src/frontend/components/SøknadsSteg/Utvidet-DinLivssituasjon/SamboerSkjema.tsx

```diff
@@ -42,16 +42,20 @@ const SamboerSkjema: React.FC<{
                 labelSpråkTekstId={samboerSpråkIder.fnrUkjent}
                 felt={samboerFelter.fnrUkjent}
             />
-            <Datovelger
-                skjema={skjema}
-                felt={samboerFelter.fødselsdato}
-                labelTekstId={samboerSpråkIder.fødselsdato}
-                disabled={samboerFelter.fødselsdatoUkjent.verdi === ESvar.JA}
-            />
-            <SkjemaCheckbox
-                labelSpråkTekstId={samboerSpråkIder.fødselsdatoUkjent}
-                felt={samboerFelter.fødselsdatoUkjent}
-            />
+            {samboerFelter.fødselsdato.erSynlig && (
```

Comment:
Dette er fiksen for https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=Tea-6111